react/the_react_ecosystem/react_router.md

```diff
@@ -417,7 +417,9 @@ const Router = () => {
 export default Router;
 ~~~
 
-So, we can simply add this `Router.jsx` component to the `main.jsx` file.
+
+So, we can simply add this `Router.jsx` component to the `Main.jsx` file.
+
```

Comment:
```suggestion
So, we can simply add this `Router.jsx` component to the `Main.jsx` file.
```

removed newly added newlines

Reply:
Added the suggestion - please let me know if everything is all right now =)